src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.TestService/System.ServiceProcess.ServiceController.TestService.csproj

```diff
@@ -50,7 +50,10 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <Content Include="System.ServiceProcess.ServiceController.TestService.runtimeconfig.json">
+    <Content Condition="'$(TargetNetCoreApp)' == 'true'" Include="System.ServiceProcess.ServiceController.TestService.runtimeconfig.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Condition="'$(TargetNetFx)' == 'true'" Include="System.ServiceProcess.ServiceController.TestService.exe.config">
```

Comment:
@danmosemsft @Anipik I believe you both worked in that area most of the time. You might be interested in the fact that you never tested against the netfx assemblies from corefx 😆